src/neuroconv/datainterfaces/behavior/video/videodatainterface.py

```diff
@@ -20,20 +20,20 @@
 from ....utils import get_schema_from_hdmf_class, get_base_schema, calculate_regular_series_rate, OptionalFilePathType
 
 
-def _check_duplicates(videos_metadata, file_paths):
+def _check_duplicates(videos_metadata: List[dict], file_paths):
     """
     Accumulates metadata for when multiple video files go in one ImageSeries container.
 
     Parameters
     ----------
-    videos_metadata: List[Dict]
+    videos_metadata: list of dict
```

Comment:
How are you thinking about this change? 
We are using types to specificy the data type in the doctstring in all other cases, why not use the type here and instead opt for a verbal description?

You feel is more clear? 
I don't have a strong opinion, but whatever we chose I feel we should add a note about this to the style guide for future reference.

Reply:
Yes, I saw that and this seems to be an edge case for me. The general form says:
```
Parameters
----------
x : type
    Description of parameter `x`.
```
Which indicates that the type should be given (as in str, float, or int). However, for a list of strings and tuples the guide does present examples that look more like a "a textual description of the type":

```
Parameters
----------
...
iterable : iterable object
shape : int or tuple of int
files : list of str
```
Above, the guide suggests "int or tuple of ints" for shape and a "list of str" for a list of files, instead of precise types which would be `Tuple[int]` and `List[str]` respectively.

Reply:
@h-mayorquin I don't think this is any less precise, just a different way of expressing the same information. I'm just trying to follow the numpy docstyle more closely. Is this OK to resolve?

Reply:
Yes, I think it is the same.